index.js

```diff
@@ -7,6 +7,7 @@ const makeValidator = require('./lib/validate')
 const { get: getSchema, set: setSchema } = require('./lib/layer-schema')
 const minimumViableDocument = require('./lib/minimum-doc')
 const generateDocument = require('./lib/generate-doc')
+const convertYaml = require('./lib/convert-yaml')
```

Comment:
Lets just inline this below.

Reply:
This one doesn't look like it has been resolved.  Can you just inline the call to the `yaml` package and remove this extra file?

Reply:
Yeah I'm not sure why is it now being reflected here, but I've included all the changes and now the index.js is shown to be "outdated". Its probably due to the way I squashed the commits

You can take a look at the "files changed" section of the PR for the changes I've made

Reply:
Yeah I see what you mean, but it still looks like the new file wasn't deleted, which is what I saw before making this comment.

Reply:
Hey @MeCode4Food, any chance you want to follow up with this? If not, I might just take over the PR myself next time I can work on this.  Just let me know if you would like to continue working on this.